### PR TITLE
Add packages hxrsnd, lightpath, pcdsdevices, pcdswidgets, and pswalker

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

These repositories are all part of a single stack of source code, so I've included them all in one.
If desirable, let me know and I can split them without issue.

cc @hhslepicka @zllentz

### CODE REVIEW

File: recipes/hxrsnd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  imports:
+    - hxrsnd
+
+about:
+  home: https://github.com/pcdshub/hxrsnd
+  doc_url: https://pcdshub.github.io/hxrsnd
+  dev_url: https://github.com/pcdshub/hxrsnd
+  license: SLAC Open
+  license_family: Other
+  license_file: LICENSE
```
Comment (hhslepicka) [inline recipes/hxrsnd/meta.yaml]: LICENSE.md but not at the MANIFEST.in maybe add the LICENSE here?

File: recipes/lightpath/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  imports:
+    - lightpath
+
+about:
+  home: https://github.com/pcdshub/lightpath
+  doc_url: https://pcdshub.github.io/lightpath
+  dev_url: https://github.com/pcdshub/lightpath
+  license: SLAC Open
+  license_family: Other
+  license_file: LICENSE
```
Comment (hhslepicka) [inline recipes/lightpath/meta.yaml]: Same as above

File: recipes/hxrsnd/meta.yaml (hunk 1 above)
Comment (klauer) [reply recipes/hxrsnd/meta.yaml]: Addressed in fba14e328

File: recipes/lightpath/meta.yaml (hunk 2 above)
Comment (klauer) [reply recipes/lightpath/meta.yaml]: Addressed in fba14e328

**Comment by hhslepicka** [comment]
I agree to be a maintainer of all recipes on this Pull Request.

**Comment by klauer** [comment]
I erroneously uploaded packages @zllentz does not want here. Will resubmit at a later date.
"not hxrsnd and lightpath and pswalker"

**Pull request closed without merging**